bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/parser/MeterModelParser.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sunspec.internal.parser;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.openhab.binding.modbus.sunspec.internal.SunSpecConstants;
+import org.openhab.binding.modbus.sunspec.internal.dto.MeterModelBlock;
+import org.openhab.io.transport.modbus.ModbusRegisterArray;
+
+/**
+ * Parser for sunspec compatible meters
+ *
+ * @author Nagy Attila Gabor - Initial contribution
+ *
+ */
+public class MeterModelParser extends AbstractBaseParser implements SunspecParser<MeterModelBlock> {
+
+    @Override
+    public MeterModelBlock parse(@NonNull ModbusRegisterArray raw) {
```

Comment:
Please remove all the excess blank lines in this method

Reply:
ok, removed blank lines